Libraries/LibJS/Tests/builtins/Array/Array.prototype.sort.js

```diff
@@ -0,0 +1,117 @@
+describe("Array.prototype.sort", () => {
```

Comment:
Please format this whole file with [prettier](https://github.com/prettier/prettier/) (notice the `.prettierrc` config file in the repository root)